build/dogfood.cmd

```diff
@@ -0,0 +1,3 @@
+@echo off
+powershell -NoLogo -NoProfile -ExecutionPolicy ByPass -NoExit %~dp0Build.ps1 -dogfood %*
+exit /b %ErrorLevel%
```

Comment:
Will I actually get those environment variables set up for me if I do dogfood.cmd?

Reply:
Seems like they'd be trapped in the powershell process

Reply:
`-NoExit` preserves the powershell context when it leaves

Reply:
I see.